### PR TITLE
added requestor and result tests

### PR DESCRIPTION
**Pull request by AaronJhaj**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: test-galasactl-ecosystem.sh [hunk 1]
```diff
@@ -672,15 +673,44 @@ function runs_get_check_raw_format_output_with_from_and_requestor {
...
+    info "Command is: $cmd"
+
+    output_file="runs-get-output.txt"
+    $cmd | tee $output_file
+
+    cat $output_file | grep "result         : $result" -q
+    rc=$?
+   
+    if [[ "${rc}" != "0" ]]; then 
+        error "Did not find any runs with result 'Passed' in output"
```
Comment (eamansour) [inline test-galasactl-ecosystem.sh:709]: Might need to replace 'Passed' with the result parameter in this message

**Review by eamansour — APPROVED** [review]
Looks good, just one minor suggestion

**Review by eamansour — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**